Blockzilla/Settings/Cells/SettingsTableViewAccessoryCell.swift

```diff
@@ -24,6 +24,12 @@ class SettingsTableViewAccessoryCell: SettingsTableViewCell {
         accessoryView = UIImageView(image: UIImage(systemName: "chevron.right"))
         tintColor = .secondaryText.withAlphaComponent(0.3)
     }
+    
+    convenience init(style: UITableViewCell.CellStyle, reuseIdentifier: String?, hasAccessoryView: Bool = true) {
```

Comment:
So by adding `hasAccessoryView` we can end up with a `SettingsTableViewAccessoryCell` with no accessory view? That doesn't sound right.

Reply:
Indeed, there was SettingsTableViewCell the cell type that should be used for this case. I updated the code. Thank you!